tests/test_urlparse.py

```diff
@@ -1,20 +1,14 @@
-# https://github.com/python/cpython/blob/40dac3272231773af0015fc35df5353783d77c4e/Lib/test/test_urlparse.py
```

Comment:
I think it would be great to leave this link, as the file is still heavily based on the original, and e.g. mentions CPython issues in comments that do not make sense without this link as a reference.